modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase

```diff
@@ -2747,6 +2747,102 @@ definition {
+	@priority = 5
+	test CanVerifyOrderItemsAreSplitOnTheOrderAdmin {
+		property custom.properties = "company.security.strangers.verify=false${line.separator}jsonws.web.service.paths.excludes=";
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with a buyer user is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the admin enables the toggle 'Show Separate Order Items' for Minium Portal channel") {
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Show Separate Order Items",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
+		}
+
+		task ("And the buyer logs in and clicks 2 times on the 'Add to cart' button for MIN55861") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			for (var count : list "1,2") {
+				Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
+
+				Click(
+					key_productName = "U-Joint",
+					locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+			}
+		}
+
+		task ("When the buyer completes the checkout") {
+			CommerceCheckout.gotoCheckout();
+
+			CommerceCheckout.checkout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				shippingMethod = "Standard");
+		}
+
+		task ("Then the admin can view the order items shown separately with the correct data on the order admin page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			CommerceOrders.openOrdersAdmin();
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
+
+			CommerceOrders.viewOrdersEntry(
+				productListPrice = "24.00",
+				productName = "U-Joint",
+				productQuantity = 1,
+				productSku = "MIN55861",
+				productTotalPrice = "24.00");
+
+			AssertTextNotEquals(
+				key_cellName1 = "U-Joint",
+				key_cellName2 = "MIN55861",
+				key_columnNumber = 7,
+				locator1 = "CommerceEntry#TABLE_CELL_BY_COLUMN_ORDER_NO_TITLE",
+				value1 = 2);
+
+			Pagination.viewResults(results = "Showing 1 to 2 of 2 entries.");
+
+			CommerceOrders.viewOrderSummary(
+				deliveryValue = "$ 15.00",
+				subtotalValue = "$ 48.00",
+				totalValue = "$ 63.00");
+		}
```

Comment:
Can you use a user with the Operations Manager role to do the assertions? This applies also to the shipments test. Thanks!